node/src/assets.rs

```diff
@@ -347,6 +348,32 @@ where
     last_id: Mutex<Option<UniqueId>>,
 }
 
+/// Iterates over a key range in column `db_col`, determined by  [`DistributedAssetStorage::<T>::make_prefix_range(my_participant_id, domain_id)`],
+/// Deletes all entries in `db_col` that evaluate `false` for `is_subset_of_active_participants(persistent_participants)`
+pub fn clean_db<T>(
+    db: &Arc<SecretDB>,
```

Comment:
References to pointers is also a bit weird. You should be able to just do

```suggestion
    db: &SecretDB,
```

Reply:
Nvm I see the `db.update()` method requires this. We should refactor this separately, but don't need to do it in this PR.